### ISSUE TITLE
`Open myPath for input` not highlighted

### ISSUE THREAD

**Issue by sancarn** [serkonda7/vscode-vba#96]
### Description and Reproduction Steps

Keywords not highlighted in:

```vb
  Open sPath For Input as #ff
    Input #ff, sContent
  Close #ff
```

**Comment by serkonda7** [comment]
Confirmed, feel free to propose a patch.
https://learn.microsoft.com/en-us/office/vba/language/reference/user-interface-help/open-statement

**Comment by DecimalTurn** [comment]
I'd suggest making a new keyword category just for I/O related keywords with scope name `keyword.io.vba`.

Here's the ones I could find:

* [Open](https://learn.microsoft.com/en-us/office/vba/language/reference/user-interface-help/open-statement)
* [Close](https://learn.microsoft.com/en-us/office/vba/language/reference/user-interface-help/close-statement)
* [Input](https://learn.microsoft.com/en-us/office/vba/language/reference/user-interface-help/inputstatement)
* [Line Input](https://learn.microsoft.com/en-us/office/vba/language/reference/user-interface-help/line-inputstatement)
* [Lock, Unlock](https://learn.microsoft.com/en-us/office/vba/language/reference/user-interface-help/lock-unlock-statements)
* [Print](https://learn.microsoft.com/en-us/office/vba/language/reference/user-interface-help/printstatement)
* [Seek](https://learn.microsoft.com/en-us/office/vba/language/reference/user-interface-help/seek-statement)
* [Width](https://learn.microsoft.com/en-us/office/vba/language/reference/user-interface-help/widthstatement)
* [Get](https://learn.microsoft.com/en-us/office/vba/language/reference/user-interface-help/get-statement)
* [Put](https://learn.microsoft.com/en-us/office/vba/language/reference/user-interface-help/writestatement)
* [Write](https://learn.microsoft.com/en-us/office/vba/language/reference/user-interface-help/writestatement)

**Comment by serkonda7** [comment]
@DecimalTurn thank you for the list. I have some time today and will implement it.

**Comment by serkonda7** [comment]
The changes are here: 33305c313305b53b89ab870d127b4375afb467ca

The test covers not all oeyeords yet.

Also some fine adjustments to the [Open statement](https://learn.microsoft.com/en-us/office/vba/language/reference/user-interface-help/open-statement) is needed.

The following parts still lack proper highlighting. But this exceeds my time for now
- [ ] mode keywords
- [ ] acces keywords
- [ ] lock keywords

**Comment by DecimalTurn** [comment]
In my opinion, it's not necessary to highlight the mode and access keywords inside the `Open` statement. Otherwise, the whole line of code is pretty much highlighted and I feel like too much highlighting is almost like no highlighting at all.

That being said, I'm not totally opposed to the idea, but that will need some more playing around with regex to make it work.

In the meantime, it would be weird that the `Input` keyword would be highlighted inside the `Open` statement and not the others (Append, Binary, Output, and Random). I tried something with a lookahead [here](https://github.com/DecimalTurn/vscode-vba/pull/5/files) to fix that. I'm just wondering if that's something that would cause issues with GitHub Linguist like we had to fix for lookbehind that were not of fixed length (https://github.com/serkonda7/vscode-vba/pull/80) 🤔.
